lib/tasks/refresh.rake

```diff
@@ -0,0 +1,25 @@
+require 'csv'
+
+# Dir.glob("#{Rails.root}/app/services/*.rb").each { |f| require f }
+
+namespace :refresh do
```

Comment:
Rake tasks seemed like the most painless way to load and reload data at this juncture.